Specs/Scene/ImplicitTileCoordinatesSpec.js

```diff
@@ -4,9 +4,30 @@ import {
 } from "../../Source/Cesium.js";
 
 describe("Scene/ImplicitTileCoordinates", function () {
+  function internalQuadtree(level, x, y) {
+    return new ImplicitTileCoordinates({
+      subdivisionScheme: ImplicitSubdivisionScheme.QUADTREE,
+      subtreeLevels: 2,
```

Comment:
could subtreeLevels be an optional argument at the end? that way you could use it everywhere in this fil.

Reply:
Good idea. I'll use the helper function everywhere except for `constructs quadtree` and `constructs octree`